Library/Homebrew/formula_installer.rb

```diff
@@ -251,10 +251,23 @@ def check_conflicts
       begin
         f = Formulary.factory(c.name)
       rescue TapFormulaUnavailableError
-        # If the formula name is in full-qualified name. Let's silently
+        # If the formula name is in fully-qualified name let's silently
```

Comment:
The comment seems broken still: “If the formula name is **a** fully-qualified name let's silently”?